spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/cassandra/CassandraReactiveRepositoriesAutoConfiguration.java

```diff
@@ -39,7 +41,7 @@
  * @see EnableReactiveCassandraRepositories
  */
 @Configuration(proxyBeanMethods = false)
-@ConditionalOnClass({ ReactiveSession.class, ReactiveCassandraRepository.class })
+@ConditionalOnClass({ ReactiveSession.class, ReactiveCassandraRepository.class, Flux.class })
```

Comment:
I am not against that change but it has nothing to do with the task at hand. Please revert that and submit it in a separate change if you think this is worth exploring.

Reply:
True, this is a mistake, I will remove.